components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -4384,6 +4384,64 @@ func TestDeleteRule(t *testing.T) {
 	}
 }
 
+func TestApplyStagedRules(t *testing.T) {
+	store, db, _ := testhelpers.SetupTestDB(t)
+	defer db.CloseDB(t)
+	defer store.Close()
+	// No project filter concerns in these tests so safe to re-use context.
```

Comment:
meaning we don't pass projects to this API or we ignore them if passed?

Reply:
We don't check the projects header at all for this API.